src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamespaceSymbol.cs

```diff
@@ -42,6 +42,14 @@ internal abstract class PENamespaceSymbol
         /// </summary>
         private Dictionary<string, TypeDefinitionHandle> _lazyNoPiaLocalTypes;
 
+        /// <summary>
+        /// Caches the last call to GetMembers(string) when that member is a namespace. Multiple types
```

Comment:
>when that member is a namespace [](http://example.com/codeflow?start=55&length=31)

This sounds confusing, "GetMembers" then "that member".

Reply:
@AlekseyTs Yes. The scenario is when `GetMembers` will be returning a single element which is a nested namespace. I clarified the comment.